### ISSUE TITLE
theaustralian.com.au

### ISSUE THREAD

**Issue by danielgruber8** [nextgens/anti-paywall#47]
Can you disable the popup 'unlock full access' that comes up when premium articles are opened?
javascript:window.lightcase.close();

**Comment by nextgens** [comment]
Yes but that requires more permissions... so it won't happen until the next major version (see #12 for why)
